tools/clang/unittests/Basic/CMakeLists.txt

```diff
@@ -8,6 +9,7 @@ add_clang_unittest(BasicTests
   FileManagerTest.cpp
   SourceManagerTest.cpp
   VirtualFileSystemTest.cpp
+  WinAdapterTest.cpp
```

Comment:
Clang/Basic is an odd place to put this. We should probably have a unit test binary at the LLVM level for DxcSupport rather than put this under Clang.